tests/integration/components/project-settings/integrations/github-project-test.js

```diff
@@ -1,5 +1,5 @@
 import Service from '@ember/service';
```

Comment:
On this file there are many instances of `assert.dom().exists().someAssertion()` we can rid off the `.exists()` assertions since every assertion after it would fail if the element being asserted on doesn't exist. This would reduce the number off assertions being made on this test module.